homeassistant/components/recorder/table_managers/event_types.py

```diff
@@ -0,0 +1,87 @@
+"""Support managing EventTypes."""
```

Comment:
The goal is to eventually break out much of the `state_attributes` and `event_data` handling from the recorder core itself to reduce the amount of code in the `Recorder` class.  This is the first module in that series which will help separate them